content/en/monitors/types/cloud_cost.md

```diff
@@ -27,23 +27,19 @@ Cost anomaly detection is in Preview. To request access, complete the form.
 {{< /callout >}}
 
 ## Overview
-Get proactive notifications on cost changes to help mitigate unexpected cloud spend. Cloud Cost Monitors help you identify cost changes quickly so you can investigate the cause. You can configure your alerts to catch unexpected changes.
+Get proactive notifications on cost changes to help mitigate unexpected cloud spend. Cloud Cost Monitors help you identify cost changes quickly so you can investigate the cause. 
 
 In order to configure Cloud Cost monitors, you need to have [Cloud Cost Management][1] set up. After it's set up, you can configure monitors to alert when costs increase or decrease.
 
 Cloud Cost monitors are evaluated with a 48 hour delayed evaluation window, because Cloud Cost data is not guaranteed to be available until 48 hours after usage. For example, a monitor with a lookback of 7 days being evaluated on January 15 examines cost data from January 6 through January 13.
 
 ## Monitor creation
 
-To create a Cloud Cost monitor in Datadog, use the main navigation: [**Monitors** --> **New Monitor** --> **Cloud Cost**][4].
-
-You can also create Cloud Cost monitors from the [Cloud Cost Explorer][2]. Click **More...** next to the Options button and select **Create monitor**.
-
-{{< img src="/monitors/monitor_types/cloud_cost/explorer.png" alt="Option to create a monitor from the Cloud Cost Explorer page" style="width:100%;" >}}
+To create a Cloud Cost monitor in Datadog, you can set up via [Terraform], from the main navigation, [**Monitors** --> **New Monitor** --> **Cloud Cost**][4], or from the [Cloud Cost Explorer][2]. 
```

Comment:
It looks like there's a missing destination link for "Terraform".

Reply:
added!